lib/govuk_security_audit/version.rb

```diff
@@ -1,3 +1,3 @@
 module GovukSecurityAudit
-  VERSION = "0.1.2"
+  VERSION = "0.1.3"
```

Comment:
I'd argue that semantically this should be 0.2.2 as it adds a backwards compatible feature, not a bug fix.